doc/motr-in-prose.md

```diff
@@ -3,7 +3,7 @@
 _« Par ma foi ! il y a plus de quarante ans que je dis de la prose sans que j'en susse rien, et je vous suis le
 plus obligé du monde de m'avoir appris cela. »_
 
-This document attempts to be a master document for the whole of Motr, 
+This document attempts to be a primary document for the whole of Motr, 
```

Comment:
Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/Seagate/cortx-motr/pullRequest?prid=9629868)
```suggestion
This document attempts to be a primary document for the whole of Motr,
```